evergreen/evergreen/urls.py

```diff
@@ -22,14 +22,12 @@
 
 from eg_app import views
 
-# from evergreen import settings
-
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", views.index, name="index"),
     path("validate", views.validate, name="validate"),
     path("updateFeed", views.updateFeed, name="updateFeed"),
-    path("uploadPost", views.uploadPost, name="uploadPost"),
+    # path("uploadPost", views.uploadPost, name="uploadPost"),
```

Comment:
If you look closely at your keyboard, you might notice there's a button at the top right that says either "backspace" or "delete" -- it's pretty nifty!

Reply:
https://medium.com/@bpnorlander/stop-writing-code-comments-28fef5272752
;)